### PR TITLE
gCAP3D: Add metadata and introduction for gCAP3D

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

Related to #42 

**PRs for new software/tutorials**

- [ ] Software metadata is complete
- [ ] The "last updated date" in the metadata page is up-to-date
- [ ] All links work
- [ ] You're listed as the author(s) in the metadata page
- [ ] Anyone who have made significant contributions (e.g., review) to
      the tutorials are listed in the meteda page

**PRs for bugs/improvements**

- [ ] You're listed as the proofreader in the metadata page

### CODE REVIEW

**Comment by core-man** [comment]
I am not sure when gCAP1.2 is released. So leave ??? or XXX.

**Comment by seisman** [comment]
> I am not sure when gCAP1.2 is released. So leave ??? or XXX.

Perhaps just saying v1.2 is enough. No need to add the date.

Apparently, he made some changes last year and recently.
image


> How do you know the latest fk is v3.3 (2015-02-03)?

I found it in the README file.

> Shall we rename it to be gCAP3D?

Yes.

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Add a labeler to any changes within folders or any subfolders
 
 # Seismological software: software name is used as the labeler
-cap: source/cap/**/*
 cps: source/cps/**/*
 fk: source/fk/**/*
+gcap: source/gcap/**/*
```
Comment (seisman) [inline .github/labeler.yml]:
```suggestion
gcap3d: source/gcap3d/**/*
```

File: source/gcap/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+gCAP 中提供了如下几个命令和脚本。
+
+- ``cap3D``\ ：用于反演地震矩张量的主程序，也是 gCAP 的核心程序
+- ``cap3D.pl``\ ：对 ``cap3D`` 的封装，一般情况下直接使用该脚本即可
+- ``depth.pl``\ ：用于读取 ``cap3D`` 输出的不同深度震源的结果，并估计最佳震源深度
+- ``mtdcmp``\ ：将地震矩张量分解为 ISO、DC 以及 CLVD 三个张量
+- ``radpttn``\ ：将 ISO、DC 以及 CLVD 三个张量合成为归一化的地震矩张量，
+  并计算 P、SV 以及 SH 波的辐射花样
+
+引用
```
Comment (seisman) [inline source/gcap/introduction.rst]: Do we want to explain the differences between cap, gcap, gcap3d, cap-tele, gcap-tele, gcap-sf et al.?

File: source/index.rst [hunk 3]
```diff
@@ -28,6 +28,14 @@
    - :doc:`/cps/index`
    - :doc:`/fk/index`
 
+.. rubric:: 震源反演：
+
+.. hlist::
+   :columns: 3
+
+   - :doc:`/cps/index`
+   - :doc:`/gcap/index`
```
Comment (seisman) [inline source/index.rst:37]:
```suggestion
   - :doc:`/gcap3d/index`
```

File: source/index.rst [hunk 4]
```diff
@@ -70,6 +78,7 @@
 
    cps/index
    fk/index
+   gcap/index
```
Comment (seisman) [inline source/index.rst:81]:
```suggestion
   gcap3d/index
```

**Comment by core-man** [comment]
> > How do you know the latest fk is v3.3 (2015-02-03)?
> 
> I found it in the README file.

If fk v3.3 is released in 2015 and updated all the time, why should
we indicate the date is 2015-02-03.

File: source/gcap/introduction.rst (hunk 2 above)
Comment (core-man) [reply source/gcap/introduction.rst]: Yes, but I think should be put on a separate page after we finish gCAP3D first.
This topic deserves more details

Comment (seisman) [reply source/gcap/introduction.rst]: OK

**Comment by seisman** [comment]
> > > How do you know the latest fk is v3.3 (2015-02-03)?
> > 
> > 
> > I found it in the README file.
> 
> If fk v3.3 is released in 2015 and updated all the time, why should
> we indicate the date is 2015-02-03.

That's a good question. Perhaps we should also remove it.

File: source/index.rst [hunk 5]
```diff
@@ -28,6 +28,14 @@
    - :doc:`/cps/index`
    - :doc:`/fk/index`
 
+.. rubric:: 震源反演：
```
Comment (seisman) [inline source/index.rst:31]:
```suggestion
.. rubric:: 震源机制反演：
```

File: source/gcap3d/index.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+:源码地址: http://www.eas.slu.edu/People/LZhu/home.html
+:作者: `Lupei Zhu <http://www.eas.slu.edu/People/LZhu/home.html>`__
+:最新版本: v1.2
+:适用平台: Linux、macOS
+:编程语言: C、Fortran、Perl
+:许可协议: MIT
+
+----
+
+:教程贡献者: `田冬冬 <https://me.seisman.info/>`__ (作者)、`姚家园 <https://github.com/core-man>`__ (审稿)
```
Comment (seisman) [inline source/gcap3d/index.rst:15]: Again, please use `（）`, instead of `()`

File: source/gcap3d/index.rst [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+----
+
+.. note::
+
+   gCAP3D v1.2 发布后，程序中的一些小问题被开发者修复，但并未更新版本号。
+   因而 gCAP3D v1.2 实际上有一些不同的小版本。
+
+   本教程仅在 Linux 和 macOS 系统下使用 v1.2（下载于 2021-01-13）测试通过，
+   其余版本或系统可能有所差异。
```
Comment (seisman) [inline source/gcap3d/index.rst:26]:
```suggestion
   本教程仅在 Linux 和 macOS 系统下使用 v1.2
   （下载于 2021-01-13，软件包中文件的最新日期为 2020-01-31）
   测试通过，其余版本或系统可能有所差异。
```

File: source/gcap3d/introduction.rst [hunk 8]
```diff
@@ -0,0 +1,40 @@
+简介
+====
+
+gCAP 可用于反演震源机制解。
```
Comment (seisman) [inline source/gcap3d/introduction.rst:4]:
```suggestion
gCAP3D 可用于反演震源机制解。
```

File: source/gcap3d/introduction.rst [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+功能
+----
+
+1. 支持反演完整的地震矩张量
+2. 支持使用三维模型下的格林函数
+
+命令及脚本
+----------
+
+gCAP 中提供了如下几个命令和脚本。
```
Comment (seisman) [inline source/gcap3d/introduction.rst:15]:
```suggestion
gCAP3D 中提供了如下几个命令和脚本：
```

File: source/gcap3d/introduction.rst [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+
+- ``cap3D``\ ：用于反演地震矩张量的主程序，也是 gCAP 的核心程序
```
Comment (seisman) [inline source/gcap3d/introduction.rst:17]:
```suggestion
- ``cap3D``\ ：用于反演地震矩张量的主程序，也是 gCAP3D 的核心程序
```

File: source/gcap3d/introduction.rst [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+- ``cap3D.pl``\ ：对 ``cap3D`` 的封装，一般情况下直接使用该脚本即可
+- ``depth.pl``\ ：用于读取 ``cap3D`` 输出的不同深度震源的结果，并估计最佳震源深度
+- ``mtdcmp``\ ：将地震矩张量分解为 ISO、DC 以及 CLVD 三个张量
```
Comment (seisman) [inline source/gcap3d/introduction.rst:20]:
```suggestion
- ``mtdcmp``\ ：将地震矩张量分解为 ISO、DC 以及 CLVD 三个部分
```

File: source/gcap3d/introduction.rst [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+- ``radpttn``\ ：将 ISO、DC 以及 CLVD 三个张量合成为归一化的地震矩张量，
```
Comment (seisman) [inline source/gcap3d/introduction.rst:21]:
```suggestion
- ``radpttn``\ ：将 ISO、DC 以及 CLVD 三个部分合成为归一化的地震矩张量，
```

File: source/gcap3d/index.rst (hunk 6 above)
Comment (core-man) [inline source/gcap3d/index.rst:15]:
```suggestion
:教程贡献者: `田冬冬 <https://me.seisman.info/>`__\ （作者）、
                    `姚家园 <https://github.com/core-man>`__\ （审稿）
```

**Review by seisman — APPROVED** [review]

**Pull request merged**